manim/mobject/geometry.py

```diff
@@ -938,15 +963,97 @@ def __init__(self, start_anchor, start_handle, end_handle, end_anchor, **kwargs)
 
 
 class Polygon(VMobject):
+    """A shape created by defining its vertices.
+
+    Parameters
+    ----------
+    vertices : :class:`list`
+        The vertices of the mobject. The first one is repeated to close the shape. Must define 3-dimensions: `[x,y,z]`
+    color : :class:`~.Colors`, optional
+        The color of the polygon.
+    kwargs : Any
+        Additional arguments to be passed to :class:`.~VMobject`
+
+    Examples
+    --------
+
+    .. manim:: PolygonExample
+        :save_last_frame:
+
+        class PolygonExample(Scene):
+            def construct(self):
+                isosceles = Polygon([-5, 1.5, 0], [-2, 1.5, 0], [-3.5, -2, 0])
+                position_list = [
+                    [4, 1, 0],  # middle right
+                    [4, -2.5, 0],  # bottom right
+                    [0, -2.5, 0],  # bottom left
+                    [0, 3, 0],  # top left
+                    [2, 1, 0],  # middle
+                    [4, 3, 0],  # top right
+                ]
+                square_and_triangles = Polygon(*position_list, color=PURPLE_B)
+                self.add(isosceles, square_and_triangles)
+    """
+
     def __init__(self, *vertices, color=BLUE, **kwargs):
         VMobject.__init__(self, color=color, **kwargs)
         # There are actually four corners, and the first one is repeated twice to form the four vertices.
         self.set_points_as_corners([*vertices, vertices[0]])
 
     def get_vertices(self):
+        """Get the vertices of a given polygon.
```

Comment:
Minor language nitpick: I think "a given polygon" is too unspecific, and we can actually just speak of "the polygon" here.

```suggestion
        """Get the vertices of the polygon.
```